kodilla-hibernate/src/test/java/com/kodilla/hibernate/manytomany/dao/CompanyDaoTestSuite.java

```diff
@@ -59,4 +64,41 @@ void testSaveManyToMany() {
             //do nothing
         }
     }
+
+    @Test
+    void testNamedQueries() {
+        //Given
+        Employee johnSmith = new Employee("John", "Smith");
+        Employee stephanieClarckson = new Employee("Stephanie", "Clarckson");
+        Employee lindaKovalsky = new Employee("Linda", "Kovalsky");
+
+        Company softwareMachine = new Company("Software Machine");
+
+        softwareMachine.getEmployees().add(johnSmith);
+        softwareMachine.getEmployees().add(stephanieClarckson);
+        softwareMachine.getEmployees().add(lindaKovalsky);
+
+        johnSmith.getCompanies().add(softwareMachine);
+        stephanieClarckson.getCompanies().add(softwareMachine);
+        lindaKovalsky.getCompanies().add(softwareMachine);
+
+        companyDao.save(softwareMachine);
+        int softwareMachineId = softwareMachine.getId();
+
+        //When
+        List<Employee> retrieveEmployeeByName = employeeDao.retrieveEmployeeByName("Kovalsky");
+        List<Company> retrieveCompaniesByName = companyDao.retrieveCompaniesBySubStr("Sof");
+
+        //Then
+        assertEquals(1, retrieveEmployeeByName.size());
```

Comment:
Asercje do poprawy: sprawdzamy, czy rzeczywiscie uzskalismy to co trzeba, a nie tylko tyle ile trzeba